src/ConstructiveSolidGeometry/PointsAndVectors.jl

```diff
@@ -35,6 +35,8 @@ end
 @inline _in_z(p::CartesianPoint, z::Real) = abs(p.z) <= z
 @inline _in_z(p::CartesianPoint, z::AbstractInterval) = p.z in z
 
+@inline _isapprox_sph_r(p::CartesianPoint{T}, radius::Real) where {T} = isapprox(hypot(p.x, p.y, p.z), radius, atol = geom_atol_zero(T))
```

Comment:
Can you check whether `norm(p)` is faster than `hypot`?

Reply:
Just did, difference is negligible